### PR TITLE
Marko/move series id into nftdetails

### PR DESCRIPTION
**Pull request by markopoloparadox**
For ergonomic reason, I moved series id out of `NFTData` and into `NFTDetails`.

Also, `Created` event now contains series id.

### CODE REVIEW

File: pallets/nfts/src/lib.rs [hunk 1]
```diff
@@ -220,8 +206,8 @@ pub mod pallet {
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     #[pallet::metadata(T::AccountId = "AccountId", T::NFTId = "NFTId")]
     pub enum Event<T: Config> {
-        /// A new NFT was created. \[nft id, owner\]
-        Created(T::NFTId, T::AccountId),
+        /// A new NFT was created. \[nft id, owner, series id\]
+        Created(T::NFTId, T::AccountId, NFTSeriesId),
```
Comment (ETeissonniere) [inline pallets/nfts/src/lib.rs:210]: Shall we just include `NFTDetails` instead?

**Review by ETeissonniere — COMMENTED** [review]
lgtm, I just think the `Created` event should include all the details rather than just the series id

File: pallets/nfts/src/lib.rs (hunk 1 above)
Comment (markopoloparadox) [reply pallets/nfts/src/lib.rs:210]: Maybe we should wait until we are done with the "capsule" feature to see what is needed or not.

Comment (ETeissonniere) [reply pallets/nfts/src/lib.rs:210]: Up to you guys - I will let @ybensacq have the last word on this

Comment (ybensacq) [reply pallets/nfts/src/lib.rs:210]: Ok we will see that in the capsule branch

**Pull request merged**